conn.go

```diff
@@ -1357,3 +1357,22 @@ func resendZkAuth(ctx context.Context, c *Conn) error {
 
 	return nil
 }
+
+
+// Synchronously gets all the ephemeral nodes matching `path` created by this session.
```

Comment:
This comment needs to match the rest and should also match Go best practices (i.e using complete sentences, starting with the method name).  Argument and returns here should be obvious and don't need documentation.

All calls are synchronous in this library so that is also not needed.  

https://golang.org/doc/effective_go.html#commentary

Also can you place this method near the rest of the request methods?  Maybe around Get?

Reply:
Sure~I will revise it according to your suggestion